spec/cloud/google_service_spec.rb

```diff
@@ -603,6 +605,18 @@
     end
   end
 
+  describe '#service_account_scope_url' do
+    it "returns the passed-in scope if it already looks like a scope URL" do
+      scope = "https://www.googleapis.com/auth/fake_scope"
+      expect(service.service_account_scope_url(scope)).to eq(scope)
+    end
+
+    it "returns a properly-formatted scope URL if a short-name or alias is provided" do
+      expect(service).to receive(:translate_scope_alias).with("scope_alias").and_return("real_scope")
```

Comment:
how does this pass? I don't see `real_scope` in the map. I'm confused how this works.